src/ansys/units/quantity_maps/bt_map.yaml

```diff
@@ -0,0 +1,98 @@
+# Quantity Map
+# ------------
+#
+# Quantity Name : Quantity unit string
```

Comment:
I think that `string` is a redundant appendage in all these. Maybe it's sufficient to say `units` on the righthand side and Quantity on the left..

Reply:
Changes added.